recipes/baked_beans.json

```diff
@@ -0,0 +1,18 @@
+{
+    "recipe_name": "Baked Beans",
+    "serves": "1",
+    "prep_time": "2 mins",
+    "cooking_time": "2 mins",
```

Comment:
Heinz says to cook it for 2.5 minutes and I can't afford a powerful microwave so lets make it 3 minutes.

Reference: https://www.kff.co.uk/heinz-baked-beans/p/11530

Reply:
Sounds good. Adjusted cooking and prep time